src/elements/common/selectors/version.js

```diff
@@ -32,8 +34,15 @@ const getVersionAction = ({ restored_at, trashed_at, version_promoted }: $Shape<
     return action;
 };
 
-const getVersionUser = ({ modified_by, restored_by, trashed_by }: $Shape<BoxItemVersion>): User => {
-    return restored_by || trashed_by || modified_by || PLACEHOLDER_USER;
+const getVersionUser = (
+    { modified_by, restored_by, trashed_by, uploader_display_name }: $Shape<BoxItemVersion>,
+    intl: IntlShape,
+): User => {
+    if (restored_by || trashed_by) return restored_by || trashed_by;
```

Comment:
I find this to be very difficult to read. We usually avoid inline conditional returns.